mesheryctl/internal/cli/root/system/export.go

```diff
@@ -0,0 +1,200 @@
+package system
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/layer5io/meshery/server/handlers"
+	"github.com/layer5io/meshery/server/models"
+	"github.com/layer5io/meshkit/models/meshmodel/core/v1alpha1"
+	"github.com/layer5io/meshkit/models/oci"
+	location "github.com/layer5io/meshkit/utils"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	save string
+	wd   string
+)
+
+var ExportCmd = &cobra.Command{
+	Use:   "export",
+	Short: "Export model",
+	Long:  `Export model to OCI format`,
+	Args:  cobra.MinimumNArgs(1),
+	Example: `
+// export a model to default path
```

Comment:
If you are requiring minimum args, then how https://github.com/meshery/meshery/pull/10530/files#diff-07b8f77c6a44e6b94b7d854945e135b15c42af9462220b05eec11a3ca85cbd47R34
is achieved?

You are adding `-o` flag and not marking the flag as required, it is sufficient. If the value of the var `save/outputPath` is empty use the default path (the current directory from which the command was invoked) otherwise the location specified by the flag.